app/views/profiles/edit.html.erb

```diff
@@ -102,6 +102,8 @@
       </div>
       <%= submit_tag t('.mfa.update'), :class => 'form__submit' %>
     <% end %>
+
+    <h2><%= link_to t('webauthn_credentials.index.title'), webauthn_credentials_path %></h2>
```

Comment:
As this stands, the user can only add webauth security keys after they have registered a TOTP authenticator device. I am not sure I understand this requirement. Shouldn't the user be able to enable mfa and only register webauth keys?